actions/plugins/setup/action.yml

```diff
@@ -23,6 +31,8 @@ runs:
       uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
       with:
         node-version: "${{ inputs.node-version }}"
+        cache: "${{ inputs.node-cache }}"
```

Comment:
do we know what happens when those inputs are empty?

Reply:
[Empty string is the default](https://github.com/actions/setup-node#usage) so it should work fine.